pkgs/applications/networking/cluster/luigi/default.nix

```diff
@@ -19,6 +19,9 @@ python3Packages.buildPythonApplication rec {
   # Requires tox, hadoop, and google cloud
   doCheck = false;
 
+  # This enables accessing modules stored in cwd
+  makeWrapperArgs = ["--prefix PYTHONPATH . :"];
```

Comment:
At first glance this approach seems a little impure.  If you're testing user-provided python modules in dev shouldn't you set PYTHONPATH on your own to your own modules, rather than have all invocations of `luigi` add `.`, wherever that may be?  Once you have your module ready to go to production, it seems like the best way might be to package it properly and add it as a dependency of luigi (perhaps via an extra plugins attribute), as we do for things like git plugins?